src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +608,13 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		if(Skript.classExists("org.bukkit.event.player.PlayerAdvancementDoneEvent")) {
+			Skript.registerEvent("Advancement done", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "[on] advancement (done|finish)")
+				.description("Called when a player does an advancement")
+				.examples("on advancement done:",
```

Comment:
```suggestion
				.examples("on advancement completion:",
```
This is just to go with the what I said just above so you can easily make changes. You can ignore this if you disagree with what I said above though.